pkg/users/account-details.js

```diff
@@ -226,7 +226,7 @@ export function AccountDetails({ accounts, groups, shadow, current_user, user })
                 </Title>
                 <EmptyStateSecondaryActions>
                     <Breadcrumb>
-                        <BreadcrumbItem onClick={() => cockpit.location.go("/")} to="#">{_("Back to accounts")}</BreadcrumbItem>
```

Comment:
Do we need to fix this in other projects as well? If they don't fix it, does it break them or just the history is gonna be broken as before?

Reply:
Just the history will be broken. We should check for this pattern also in the external projects.

Reply:
I have checked c-machines, c-podman, and starter-kit. None of them uses this pattern.